### PR TITLE
range-v3: update to version 0.9.0

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

**Comment by alkino** [comment]
The patch has been proposed upstream: https://github.com/ericniebler/range-v3/pull/1374

File: var/spack/repos/builtin/packages/range-v3/package.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class RangeV3(CMakePackage):
     git      = "https://github.com/ericniebler/range-v3.git"
 
     version('develop', branch='master')
+    version('0.9.0', sha256='ce6e80c6b018ca0e03df8c54a34e1fd04282ac1b068cd39e902e2e5201ac117f')
```
Comment (alkino) [inline var/spack/repos/builtin/packages/range-v3/package.py]: oups I need to fix the sha256

**Review by matz-e — APPROVED** [review]
LGTM

**Comment by alkino** [comment]
retest this please Jenkins

**Comment by alkino** [comment]
retest this please Jenkins

**Comment by alkino** [comment]
retest this please Jenkins

**Pull request merged**